### ISSUE TITLE
Error when install plugin redmine_git_hosting

### ISSUE THREAD

**Issue by skunkec** [jbox-web/redmine_git_hosting#702]
whene i executed in cmd this code "C:\Bitnami\redmine-3.4.3-1\apps\redmine\htdocs>bundle install"

i get error
```
[!] There was an error parsing `Gemfile`:
[!] There was an error parsing `Gemfile`: You cannot specify the same gem twice with different version requirements.
You specified: redcarpet (~> 3.4.0) and redcarpet (~> 3.3.2). Bundler cannot continue.

   from C:/Bitnami/redmine-3.4.3-1/apps/redmine/htdocs/plugins/redmine_git_hosting/Gemfile:7
   -------------------------------------------
    Rack parser for Hrack
   gem 'rack-parser', require: 'rack/parser'
  Gitolite Admin repository management
   -------------------------------------------
. Bundler cannot continue.

   from C:/Bitnami/redmine-3.4.3-1/apps/redmine/htdocs/Gemfile:111
   -------------------------------------------
     eval_gemfile file
   end
   source 'https://rubygems.org'
   -------------------------------------------
```
## any suggestions???
I tried bundle install --without development test / there is no difference

**Comment by bdbjack** [comment]
I have also experienced this issue under the following environment:


```
Environment:
  Redmine version                3.4.4.stable.17173
  Ruby version                   2.3.1-p112 (2016-04-26) [x86_64-linux-gnu]
  Rails version                  4.2.8
  Environment                    production
  Database adapter               PostgreSQL
SCM:
  Subversion                     1.9.3
  Git                            2.7.4
  Filesystem                     
Redmine plugins:
  bitbucket_reference_redmine    0.1.3
  clipboard_image_paste          1.12
  periodictask                   3.2.1
  redmine_agile                  1.4.5
  redmine_banner                 0.1.2
  redmine_checklists             3.1.10
  redmine_code_review            0.9.0
  redmine_impersonate            1.0.0
  redmine_issue_favicon          0.0.3
  redmine_issues_tree            0.0.10
  redmine_lightbox2              0.4.3
  redmine_messenger              1.0.1
```


Got error:

```
[!] There was an error parsing `Gemfile`: 
[!] There was an error parsing `Gemfile`: You cannot specify the same gem twice with different version requirements.
You specified: redcarpet (~> 3.4.0) and redcarpet (~> 3.3.2). Bundler cannot continue.

 #  from /var/lib/redmine/plugins/redmine_git_hosting/Gemfile:7
 #  -------------------------------------------
 #  gem 'gitlab-grack', git: 'https://github.com/jbox-web/grack.git', require: 'grack', branch: 'fix_rails4'
 >  gem 'redcarpet', '~> 3.3.2'
 #  
 #  -------------------------------------------
. Bundler cannot continue.

 #  from /var/lib/redmine/Gemfile:113
 #  -------------------------------------------
 #  Dir.glob File.expand_path("../plugins/*/{Gemfile,PluginGemfile}", __FILE__) do |file|
 >    eval_gemfile file
 #  end
 #  -------------------------------------------

```

**Comment by catsem** [comment]
This has already been fixed in commit e4c66dce92a8bfb8a636082df1089a56a87ad034
Strange that it could not find it's way into the last release... maybe there were some old files in the branch the autor used for the release version 1.2.3

Fix:
Since the newer redmine versions (at least >3.4) require redcarpet >3.4.0 just delete the line "gem 'redcarpet', '~> 3.3.2'" from ../redmine/plugins/redmine_git_hosting/Gemfile

**Comment by n-rodriguez** [comment]
> Strange that it could not find it's way into the last release

It's coming ;) BTW this plugin might not work on Windows due to libgit and libssh2 dependencies.
